README.md

```diff
@@ -117,9 +116,6 @@ async function uploadWithTurbo() {
   } catch (error) {
     // upload failed
     console.error('Failed to upload data item!', error);
-  } finally {
```

Comment:
Thoughts behind excision here?

Reply:
i removed the inital balance check to simplify the quick start example so the finally is not relevant. i felt all the API calls where overwhelming and can instead be communicated via diagrams or more robust example workflows in the docs